apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -502,19 +507,25 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
             });
     }
 
-    private setAllowedContentTypes(pageState: DotPageRenderState): void {
+    private setAllowedContentTypes(
```

Comment:
```suggestion
    private getAllowedContentTypes(
```

No more than 2 params is recommend an object.